kpatch-build/create-diff-object.c

```diff
@@ -771,6 +771,88 @@ static bool kpatch_line_macro_change_only(struct section *sec)
 
 	return true;
 }
+
+#elif __s390x__
+
+static int _insn_get_length(unsigned char *inst)
+{
+	int length;
+
+	switch (inst[0] >> 6) {
+		case 0b00:
+			length = 2;
+			break;
+		case 0b01:
+		case 0b10:
+			length = 4;
+			break;
+		case 0b11:
+			length = 6;
+			break;
+	}
+
+	return length;
+}
```

Comment:
Minor style suggestion: we can save a few lines by skipping the local variable and breaks without sacrificing readability:

``` C
static int _insn_get_length(unsigned char *inst)
{
	switch (inst[0] >> 6) {
	case 0b00:
		return 2;
	case 0b01:
	case 0b10:
		return 4;
	case 0b11:
		return 6;
	}
}
```